File: GestionBD/MissionEmploye.vb
Public Class MissionEmploye

    <Newtonsoft.Json.JsonProperty("codeemploye")>
    Public Property codeemploye() As String

    <Newtonsoft.Json.JsonProperty("codemission")>
    Public Property codemission() As String

    <Newtonsoft.Json.JsonProperty("datemission")>
    Public Property Datemission() As String

    <Newtonsoft.Json.JsonProperty("codepartenaire")>
    Public Property codepartenaire() As String

    <Newtonsoft.Json.JsonProperty("heuredepart")>
    Public Property heuredepart() As String

    <Newtonsoft.Json.JsonProperty("heurearrive")>
    Public Property heurearrive() As String

    <Newtonsoft.Json.JsonProperty("objet")>
    Public Property objet() As String

    <Newtonsoft.Json.JsonProperty("adresse")>
    Public Property adresse() As String

    <Newtonsoft.Json.JsonProperty("prixu")>
    Public Property prixu() As String

    <Newtonsoft.Json.JsonProperty("unite")>
    Public Property unite() As String

    <Newtonsoft.Json.JsonProperty("info1")>
    Public Property info1() As String


    <Newtonsoft.Json.JsonProperty("info2")>
    Public Property info2() As String


    <Newtonsoft.Json.JsonProperty("info3")>
    Public Property info3() As String

    Public Overloads Overrides Function Equals(obj As Object) As Boolean
        If obj Is Nothing OrElse Not Me.GetType() Is obj.GetType() Then
            Return False
        End If

        Dim missionemploye = CType(obj, MissionEmploye)
        Return Me.codepartenaire.Equals(missionemploye.codepartenaire) And Me.Datemission.Equals(missionemploye.codemission) And Me.codemission.Equals(missionemploye.codemission) And Me.codeemploye.Equals(missionemploye.codeemploye) And Me.heuredepart.Equals(missionemploye.heuredepart) And Me.heurearrive.Equals(missionemploye.heurearrive) And Me.objet.Equals(missionemploye.objet) And Me.adresse.Equals(missionemploye.adresse) And Me.prixu.Equals(missionemploye.prixu) And Me.unite.Equals(missionemploye.unite) And Me.info1.Equals(missionemploye.info1) And Me.info2.Equals(missionemploye.info2) And Me.info3.Equals(missionemploye.info3)
    End Function

End Class

File: AndroidApp-GRH-MC/app/src/main/java/com/example/mcgrhpfe_2021/MissionEmploye.java
package com.example.mcgrhpfe_2021;


import android.content.pm.PermissionInfo;

public class MissionEmploye {
    private String adresse ;
    private String codeemploye ;
    private String codemission ;
    private String codepartenaire ;
    private String datemission ;
    private String heurearrive ;
    private String heuredepart ;
    private String objet ;
    private String prixu ;
    private String unite ;
    private String info1 ;
    private String info2 ;
    private String info3 ;

    MissionEmploye()
    {


    }

    public MissionEmploye(String adresse, String codeemploye, String codemission, String codepartenaire, String datemission, String heurearrive, String heuredepart, String objet, String prixu, String unite, String info1, String info2, String info3) {
        this.adresse = adresse;
        this.codeemploye = codeemploye;
        this.codemission = codemission;
        this.codepartenaire = codepartenaire;
        this.datemission = datemission;
        this.heurearrive = heurearrive;
        this.heuredepart = heuredepart;
        this.objet = objet;
        this.prixu = prixu;
        this.unite = unite;
        this.info1 = info1;
        this.info2 = info2;
        this.info3 = info3;
    }

    public String getInfo1() {
        return info1;
    }

    public void setInfo1(String info1) {
        this.info1 = info1;
    }

    public String getInfo2() {
        return info2;
    }

    public void setInfo2(String info2) {
        this.info2 = info2;
    }

    public String getInfo3() {
        return info3;
    }

    public void setInfo3(String info3) {
        this.info3 = info3;
    }

    public String getAdresse() {
        return adresse;
    }

    public void setAdresse(String adresse) {
        this.adresse = adresse;
    }

    public String getCodeemploye() {
        return codeemploye;
    }

    public void setCodeemploye(String codeemploye) {
        this.codeemploye = codeemploye;
    }

    public String getCodemission() {
        return codemission;
    }

    public void setCodemission(String codemission) {
        this.codemission = codemission;
    }

    public String getCodepartenaire() {
        return codepartenaire;
    }

    public void setCodepartenaire(String codepartenaire) {
        this.codepartenaire = codepartenaire;
    }

    public String getDatemission() {
        return datemission;
    }

    public void setDatemission(String datemission) {
        this.datemission = datemission;
    }

    public String getHeurearrive() {
        return heurearrive;
    }

    public void setHeurearrive(String heurearrive) {
        this.heurearrive = heurearrive;
    }

    public String getHeuredepart() {
        return heuredepart;
    }

    public void setHeuredepart(String heuredepart) {
        this.heuredepart = heuredepart;
    }

    public String getObjet() {
        return objet;
    }

    public void setObjet(String objet) {
        this.objet = objet;
    }

    public String getPrixu() {
        return prixu;
    }

    public void setPrixu(String prixu) {
        this.prixu = prixu;
    }

    public String getUnite() {
        return unite;
    }

    public void setUnite(String unite) {
        this.unite = unite;
    }
}
